src/plugins/intel_npu/src/plugin/npuw/compiled_model.hpp

```diff
@@ -66,6 +67,11 @@ class CompiledModel : public ov::npuw::ICompiledModel {
 
     void report_io() const;
 
+    void serialize(std::ostream& stream) const;
+    static std::shared_ptr<CompiledModel> deserialize(std::istream& stream,
+                                                      const std::shared_ptr<const ov::IPlugin>& plugin,
+                                                      const ov::AnyMap& properties);
```

Comment:
What's the reason to have properties here, do they control anything?

Reply:
Nope, removed